packages/binaryen/binaryen.0.11.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for Binaryen"
+maintainer: "oscar@grain-lang.org"
+authors: "Oscar Spencer"
+license: " Apache-2.0"
+homepage: "https://github.com/grain-lang/binaryen.ml"
+bug-reports: "https://github.com/grain-lang/binaryen.ml/issues"
+depends: [
+  "ocaml" {>= "4.12"}
+  "dune" {>= "2.9.1"}
+  "dune-configurator" {>= "2.9.1"}
+  "js_of_ocaml" {>= "3.10.0"}
+  "js_of_ocaml-ppx" {>= "3.10.0"}
+  "js_of_ocaml-compiler" {>= "3.10.0"}
+  "libbinaryen" {>= "101.0.1" & < "102.0.0"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+dev-repo: "git+https://github.com/grain-lang/binaryen.ml.git"
```

Comment:
```suggestion
available: arch != "arm32" & arch != "x86_32"
dev-repo: "git+https://github.com/grain-lang/binaryen.ml.git"
```

Reply:
CI on those systems fails with to compile with unrepresentable integer literal

Reply:
Please don't apply that. The whole reason for this work is to support Mac M1. I've tested this and it should be working

Reply:
Sorry, I read that wrong, it's just 32bit systems?

Reply:
Yes:
```
#       ocamlc js/.binaryen_js.objs/byte/binaryen__Module.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.13/bin/ocamlc.opt -w -40 -g -bin-annot -I js/.binaryen_js.objs/byte -I /home/opam/.opam/4.13/lib/js_of_ocaml -I /home/opam/.opam/4.13/lib/uchar -I virtual/.binaryen.objs/byte -intf-suffix .ml -no-alias-deps -open Binaryen__binaryen_js__ -o js/.binaryen_js.objs/byte/binaryen__Module.cmo -c -impl js/module.pp.ml)
# File "js/module.ml", line 77, characters 17-27:
# 77 |   split_features 0x80000000
#                       ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
#       ocamlc src/.binaryen_native.objs/byte/binaryen__Module.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.13/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.binaryen_native.objs/byte -I /home/opam/.opam/4.13/lib/libbinaryen/c -I virtual/.binaryen.objs/byte -intf-suffix .ml -no-alias-deps -open Binaryen__binaryen_native__ -o src/.binaryen_native.objs/byte/binaryen__Module.cmo -c -impl src/module.ml)
# File "src/module.ml", line 88, characters 17-27:
# 88 |   split_features 0x80000000
#                       ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
#
```

Reply:
64 bit systems from what I can see from the CI are fine

Reply:
Sounds good. I should ask the other team members about that literal. 🤔